src/pygnssutils/gnssntripclient.py

```diff
@@ -360,24 +361,54 @@ def _formatGGA(self) -> tuple:
         # time will default to current UTC
 
         try:
-            lat, lon, alt, sep = self._app_get_coordinates()
+            lat, lon, alt, sep, siv, rtk, age = self._app_get_coordinates()
             lat = float(lat)
             lon = float(lon)
+            stype = 1
+            if rtk == 1:
+                stype = 5
+            elif rtk == 2:
+                stype = 4
+            dAge = 0
+            if age == 1:

```

Comment:
I would be incined to reflect the maximum rather than 'average' age represented by the age code - see for example https://github.com/semuconsulting/PyGPSClient/blob/master/src/pygpsclient/helpers.py#L517